### DISCUSSION TITLE
Doubts on FreeRtos port

### DISCUSSION THREAD

**Discussion by lonejack** [cesanta/mongoose#2563] [Ask questions, get answers] [answered]
The CM4F port is adopted also on H7 (sub)projects. Is it rigth?
Thx
C.

/home/claudio/extension/stm32/mongoose/examples/stm32/nucleo-f429zi-make-freertos-builtin/Makefile
  13,41: SOURCES += FreeRTOS-Kernel/portable/GCC/ARM_CM4F/port.c
  15,42: CFLAGS += -IFreeRTOS-Kernel/portable/GCC/ARM_CM4F -Wno-conversion

/home/claudio/extension/stm32/mongoose/examples/stm32/**nucleo-h723zg-make-freertos-builtin**/Makefile
  13,41: SOURCES += FreeRTOS-Kernel/portable/GCC/ARM_CM4F/port.c
  15,42: CFLAGS += -IFreeRTOS-Kernel/portable/GCC/ARM_CM4F -Wno-conversion
 
/home/claudio/extension/stm32/mongoose/examples/stm32/**nucleo-h743zi-make-freertos-builtin**/Makefile
  13,41: SOURCES += FreeRTOS-Kernel/portable/GCC/ARM_CM4F/port.c
  15,42: CFLAGS += -IFreeRTOS-Kernel/portable/GCC/ARM_CM4F -Wno-conversion

**Comment by scaprile** [comment, marked as answer]
From the very same file where you copied that, one line above:
https://github.com/cesanta/mongoose/blob/b8bfa5565b10000b38a10780ad0af72babbb662d/examples/stm32/nucleo-h723zg-make-freertos-builtin/Makefile#L11
https://github.com/cesanta/mongoose/blob/b8bfa5565b10000b38a10780ad0af72babbb662d/examples/stm32/nucleo-h743zi-make-freertos-builtin/Makefile#L11

Of course it is "right", our examples do work and are tested on a CI/CD pipeline on every change.

From our tutorials:
https://mongoose.ws/documentation/tutorials/stm32/all-make-freertos-builtin/#freertos-integration

Please follow the guidelines in the [documentation](https://mongoose.ws/documentation/),  [examples](https://github.com/cesanta/mongoose/tree/master/examples/) and [tutorials](https://mongoose.ws/documentation/#tutorials-and-examples) available.

**Reply by lonejack** [reply]
You're rigth! Thank you...
